pandas/core/indexes/accessors.py

```diff
@@ -243,3 +243,11 @@ class CombinedDatetimelikeProperties(DatetimeProperties, TimedeltaProperties):
     # the Series.dt class property. For Series objects, .dt will always be one
     # of the more specific classes above.
     __doc__ = DatetimeProperties.__doc__
+
+    @classmethod
+    def _make_accessor(cls, data):
+        try:
+            return maybe_to_datetimelike(data)
+        except Exception:
+            raise AttributeError("Can only use .dt accessor with datetimelike "
+                                 "values")
```

Comment:
Sure.  I'll take care of this and anything else that piles up this evening.  To the extent that we can exercise self-control, I'm fervently hoping to keep the nitpicking back in #17042.